pkg/controller/kubedirectorcluster/members.go

```diff
@@ -280,10 +330,83 @@ func handleReadyMembers(
 				)
 				return
 			}
+
+			memberVersion := m.StateDetail.LastConnectionVersion
+
+			shared.LogInfo(
+				reqLogger,
+				cr,
+				shared.EventReasonCluster,
+				fmt.Sprintf("memberVersion for pod %s is %d", m.Pod, memberVersion),
+			)
+
+			if memberVersion < connectionsVersion {
+				shared.LogInfo(
+					reqLogger,
+					cr,
+					shared.EventReasonCluster,
+					fmt.Sprintf("--reconnect will be called here for pod : %s", m.Pod),
+				)
+
+				//*****
+				// currVersion = cr.Annotations["cluster"]
+
+				// if cr.Annotations[m.Pod] == nil
+				// 		then, initialize new key and value cr.Annotations[m.Pod] = currVersion
+
+				// if cr.Annotations[m.Pod] < currVersion
+				// it means that the Pod wasn't updated after conns changed
+				// call --reconnect and cr.Annotations[m.Pod]++.
+				// We don't do cr.Annotations[m.Pod]=currVersion because if the version changes when this is being updated,
+				// we want to call --reconnect again
+
+				// if cr.Annotations[m.Pod] > currVersion
+				// We dont want such a scenario, error out
+
+				//if cr.Annotations[m.Pod] == currVersion
+				// no action, continue
+
+				//*****
+
+				containerID := m.StateDetail.LastConfiguredContainer
+				cmd := fmt.Sprintf(appPrepConfigRunCmdTest, containerID)
+
+				shared.LogInfo(
+					reqLogger,
+					cr,
+					shared.EventReasonCluster,
+					fmt.Sprintf("Executing command %s", cmd),
+				)
+				cmdErr := executor.RunScript(
+					reqLogger,
```

Comment:
is the configmeta successfully updated by now?  how do we know?

Reply:
we know that because if there is an error in configmeta, we [return](https://github.com/saurabh-jogalekar/kubedirector/blob/10bfba8e9d1a5f76a72b29a6b6f8dd33a0da771f/pkg/controller/kubedirectorcluster/members.go#L331)